src/main/resources/static/js/tutorial/tutorial-header.js

```diff
@@ -0,0 +1,206 @@
+    </div>
+    <p id="tooltip_content"></p>
+    <div>
+    </div>
+  </div>`
+);
+
+$("#proj_list").prepend($button1);
+
+// body 태그 내에 vail, tooltip들 넣기
+$("body").append($vailUp);
+$("body").append($vailDown);
+$("body").append($vailLeft);
+$("body").append($vailRight);
+$("body").append($tooltip);
+
+// vail, tooltip 기본 style 설정
+const vails = [$vailUp, $vailDown, $vailLeft, $vailRight];
+const property_vail = {
+  display: "none",
+  background: "rgba(0, 0, 0, 0.2)",
+  position: "absolute",
+  "z-index": 100,
+};
+for (const vail of vails) vail.css(property_vail);
+$tooltip.css({
+  display: "none",
+  background: "white",
+  padding: "10px",
+  "z-index": 110,
+  "border-radius": "10px",
+  "box-shadow": "rgb(0 0 0 / 25%) 0px 0px 6px 2px",
+  "flex-direction": "column",
+});
+$("#tooltip_title_container").css({
+  display: "flex",
+  "justify-content": "space-between",
+  "align-items": "center",
+});
+$("#tooltip_title").css({
+  display: "inline-block",
+  "font-weight": 600,
+});
+$("#tooltip_content").css({
+  "margin-top": "5px",
+  "margin-bottom": "15px",
+});
+
+// 하이라이트할 부분 설정
+let elem_index = 0;
+let elem_highlight = array_highlights[elem_index];
+
+// "Project List" 버튼을 눌렀을 때
+// 튜토리얼을 처음부터 보여줌
+//$("#btn_project_list").on("click", () => {
+//  elem_index = 0;
+//  elem_highlight = array_highlights[elem_index];
+//  locate_vails();
+//  show_vails();
+//  locate_tooltip();
+//  show_tooltip();
+//});
+$("#continue_tutorial").on("click", () => {
+  elem_index = 0;
+  elem_highlight = array_highlights[elem_index];
+  locate_vails();
+  show_vails();
+  locate_tooltip();
+  show_tooltip();
+});
+// 브라우저 창 크기를 바꿀 시
+// vail, tooltip의 위치 조절
+$(window).resize(() => {
+  locate_vails();
+  locate_tooltip();
+});
+
+// 튜토리얼 - 이전, 다음, 닫기 버튼을 눌렀을 때 적절히 처리
+$("#button_prev").on("click", handle_click_prev);
+$("#button_next").on("click", handle_click_next);
+$("#button_close_tooltip").on("click", hide_vails_and_tooltips);
+
+//////////////////////////////// FUNCTIONS ////////////////////////////////
+
+// 4개 vail 각각의 위치, 크기 설정
+function locate_vails() {
+  let offset = elem_highlight.offset();
+
+  $vailUp.css({
+    top: 0,
+    left: 0,
+    width: "100%",
+    height: `${offset.top}`,
+  });
+  $vailDown.css({
+    top: `${offset.top + elem_highlight.outerHeight(true)}px`,
+    left: 0,
+    width: "100%",
+    height: `calc(100% - ${offset.top + elem_highlight.outerHeight(true)}px)`,
+  });
+  $vailLeft.css({
+    top: `${offset.top}px`,
+    left: 0,
+    width: `${offset.left}px`,
+    height: `${elem_highlight.outerHeight(true)}`,
+  });
+  $vailRight.css({
+    top: `${offset.top}px`,
+    left: `${offset.left + elem_highlight.outerWidth(true)}px`,
+    width: `calc(100% - ${offset.left + elem_highlight.outerWidth(true)}px)`,
+    height: `${elem_highlight.outerHeight(true)}`,
+  });
+}
+
+// vail 모두 보이게 설정
+function show_vails() {
+  for (const vail of vails) vail.css("display", "block");
```

Comment:
How about changing it like below? It would be more readable.

```js
for (const vail of vails) {
    vail.css("display", "block");
}
```

Reply:
Thanks! However, when we applied your code, it didn't work well 😭

Reply:
@myway00 I mean, for-loop with curly braces is more readable.

Reply:
https://github.com/fosslight/fosslight/pull/672/commits/c7a55636eed9c7231c110ec400275df0a745bb34 I applied it ! Thanks~🤗